packages/pipboard/public/index.html

```diff
@@ -6,6 +6,7 @@
   <meta http-equiv="x-ua-compatible" content="ie=edge,chrome=1">
   <meta name="viewport" content="width=device-width">
   <title>Pipcook Board</title>
+  <script src="https://cdn.jsdelivr.net/npm/@tensorflow/tfjs/dist/tf.min.js"> </script>
```

Comment:
Shall we use npm way to install tfjs? just to be consistent to any other dependencies

Reply:
Added @tensorflow/tfjs into dependencies of pipboard.